Tests/Tests/RequestSpec.swift

```diff
@@ -51,6 +59,21 @@ class RequestSpec: QuickSpec {
           expect(result.value?.count).to(equal(2))
           expect(result.value?.first?.name).to(equal("giles"))
         }
+        
+        context("when the ResponseType is Void") {
+          it("should have a nil Result value") {
+            let request = VoidResponseRequest()
+            let json = JSON.parse([])
+            let result = request.parse(json)
+            
+            switch result {
+            case .Success:
+              XCTAssert(true)
+            default:
+              XCTFail("Unexpected Failure")
+            }            
```

Comment:
Could be useful since it sounds like you're planning on moving that stuff to it's own project anyway.

I can add if you like.